code/address-lookup-table/create/main.en.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Connection,
+  Keypair,
+  Transaction,
+  AddressLookupTableProgram,
+} from "@solana/web3.js";
+import * as bs58 from "bs58";
+
+// connection
+const connection = new Connection("https://api.devnet.solana.com");
+
+// G2FAbFQPFa5qKXCetoFZQEvF9BVvCKbvUZvodpVidnoY
+const feePayer = Keypair.fromSecretKey(
+  bs58.decode(
+    "4NMwxzmYj2uvHuq8xoqhY8RXg63KSVJM1DXkpbmkUY7YQWuoyQgFnnzn6yo3CMnqZasnNPNuAT2TLwQsCaKkUddp"
+  )
+);
+
+(async () => {
+  let [instruction, addressLookupTablePubkey] =
+    AddressLookupTableProgram.createLookupTable({
+      authority: feePayer.publicKey,
+      payer: feePayer.publicKey,
+      recentSlot: await connection.getSlot(),
```

Comment:
not quite sure at this part cuz sometimes I will get an error like
`xxxx is not a recent slot`

maybe get SlotHashs system var is the most stable way to avoid it 🤔